index.rst

```diff
@@ -8,11 +8,12 @@ to Python_ -- for both new and experienced contributors.  It is
 that maintains Python.  We welcome your contributions to Python!
 
 
-Quick Start
+Quick Reference
 -----------
```

Comment:
This line needs to be extended to avoid a warning (which Travis treats as an error).